platform_versioned_docs/version-23.2.0/enterprise/advanced-topics/use-iam-role.mdx

```diff
@@ -13,22 +13,130 @@ AWS-based customers can configure Tower to interact with other AWS Services like
 
 ## Configure the Tower IAM policy
```

Comment:
Arg. I commented on 23.1.0 since it was the first file presented. 

My concerns about Data Explorer and Data Studio are obvioiusly only relevant to the appropriate Tower versions that support it but the rest holds true.